karate-core/src/main/java/com/intuit/karate/core/ScenarioEngine.java

```diff
@@ -598,6 +599,7 @@ private void httpInvokeOnce() {
             long responseTime = endTime - startTime;
             String message = "http call failed after " + responseTime + " milliseconds for url: " + httpRequest.getUrl();
             logger.error(e.getMessage() + ", " + message);
+            logger.error(ExceptionUtils.getStackTrace(e));
```

Comment:
I'm not in favor of this. this is one of the most common paths in karate and I've tried my best to minimize "log noise" over the last few years. as long as the core message comes through as a one-liner I think we should leave it as is. sorry if that sounds harsh but I really don't want to make this kind of a change without being sure. 

if you insist, we can put this into the 1.4.0 series and under DEBUG

Reply:
Happy to put it into TRACE or DEBUG (with surrounding if). Reason why I was looking for it is a team that has a weird issue of socket connection reset and after triaging all bits and pieces on the server side it just seems to be unrelated and a client issue (code or machine). Right now we loose visibility of this stacktrace at this catch block and doesn't offer much more information to triage from there.

Reply:
ok `TRACE` will work for now